src/blocks/BoardListBlock.tsx

```diff
@@ -1,38 +1,211 @@
+                defaultBorderColor={color}
+                link={{ onClick: _onMuted }}
+              />
+            </div>
+              <ActionLink link={link}>
+                Go to Board{" "}
+                <FontAwesomeIcon
+                  icon={faArrowAltCircleRight}
+                  size="lg"
+                  color={DefaultTheme.MENU_ITEM_ICON_COLOR}
+                />{" "}
+              </ActionLink>
+          
+          </div>
         </div>
-      </DropdownMenu>
+      </div>
     )}
-    {_selected && <p className="item-details">{description}</p>}
     <style jsx>{`
-      .board-list-options {
-        padding: 0 1rem;
+      * {
+        box-sizing: border-box;
+        text-align: center;
+        border: none;
+      }
+      // TODO: see about switching to css.resolve
+      :global(.insanity) {
+        min-width: 45px;
+        min-height: 100%;
+        border-radius: 0 12px 12px 0;
+        background-color: ${DefaultTheme.MENU_ITEM_ICON_BACKGROUND_COLOR};
+      }
+      .item-details {
+        grid-column: -1 / 1;
+        border-radius: 12px;
+        background-color: ${DefaultTheme.BOARD_FILTER_BACKGROUND};
+        color: ${DefaultTheme.MENU_ITEM_ICON_COLOR};
+        justify-self: stretch;
+        align-self: stretch;
+        max-width: 100%;
+        padding: 15px;
+        display: grid;
+        grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
+        gap: 20px;
+      }
+      .toggle-buttons {
+        display: flex;
+        justify-content: center;
+        gap: 1rem;
+      }
+      .board-info {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        gap: 1rem;
+        padding: 1rem;
+      }
+      .board-info-footer {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
       }
-      .board-list-options:hover {
+      .item-summary {
+        display: flex;
+        justify-content: space-between;
+        justify-self: stretch;
+        align-self: center;
+        border-radius: 15px;
+        background-color: ${DefaultTheme.BOARD_MENU_ITEM_BACKGROUND};
+      }
+      .slug-container {
+        flex-grow: 1;
+        background-color: transparent;
+        max-width: calc(100% - 60px);
+        padding: 0 5px;
+        display: flex;
+        flex-grow: 1;
+        justify-content: center;
+        align-items: center;
         cursor: pointer;
       }
+      .with-options {
+        max-width: calc(100% - 60px - 45px);
+      }
+      .slug,
+      .big-slug {
+        color: ${DefaultTheme.PINNED_BAR_TEXT_COLOR};
+        font-size: var(--font-size-large);
+        font-weight: 500;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+        flex-grow: 1;
+      }
+      .big-slug {
+        white-space: unset;
+      }
+      .muted .slug {
+        text-decoration: line-through;
+        color: #c7c7c7;
```

Comment:
I think there might be a theme variable fort his, feel free to leave if there isn't